ui/src/app/settings/views/Configuration/Security/TLSEnabled/TLSEnabled.tsx

```diff
@@ -20,7 +19,8 @@ export type TLSEnabledValues = {
 export enum Labels {
   NotificationCheckbox = "Notify the certificate is due to expire in...",
   Interval = "Days",
-  IntervalRangeError = "Notification interval must be between 0 and 90 days.",
+  IntervalRangeError = "Notification interval must be between 1 and 90 days.",
```

Comment:
Apparently `Computed values are not permitted in an enum with string valued members.` D:

Reply:
I think for strings that have computed values it would be OK to not have the string defined in an enum (even though in this case it would use values that are also enums). I think that computing a string using the min/max values here might be less error prone.